src/main/java/Deadlines.java

```diff
@@ -0,0 +1,43 @@
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Deadlines extends Task {
+    private final String by;
+    private boolean savedBefore;
```

Comment:
Perhaps the use of isSavedBefore would be more suitable as a boolean variable name?